examples/gpi_pd_minecart.py

```diff
@@ -57,79 +47,17 @@ def make_env():
         dynamics_rollout_starts=5000,
         dynamics_rollout_len=1,
         real_ratio=0.5,
-        log=False,
+        log=True,
         project_name="MORL Baselines - MineCart",
         experiment_name=f"{algo} + gpid-pd={gpi_pd} g={g}",
```

Comment:
We should also be careful in the experiment names when launching the benchmarks, we want that to be filterable so it needs to be formatted the same way for each algo